tests/foreman/ui_airgun/test_config_group.py

```diff
@@ -0,0 +1,62 @@
+"""Test class for Config Groups UI
+
+:Requirement: Config Group
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: Low
+
+:Upstream: No
+"""
+
+
+from fauxfactory import gen_string
+from nailgun import entities
+
+from robottelo.decorators import fixture, tier2, upgrade
+
+
+@fixture(scope='module')
+def module_puppet_class():
+    return entities.PuppetClass().create()
+
+
+@tier2
+@upgrade
+def test_positive_end_to_end(session, module_puppet_class):
+    """Perform end to end testing for config group component
+
+    :id: 3ac47175-1239-4481-9ae2-e31980fb6607
+
+    :expectedresults: All expected CRUD actions finished successfully
+
+    :CaseLevel: Integration
+
+    :CaseImportance: High
+    """
+    name = gen_string('alpha')
+    new_name = gen_string('alpha')
+    with session:
+        # Create new config group
+        session.configgroup.create({
+            'name': name,
+            'classes.assigned': [module_puppet_class.name],
+        })
```

Comment:
One query here: 

1. Will this test even work for satellite having 0 configs, please consider below image?

![Config Groups](https://user-images.githubusercontent.com/3190629/55478401-473fd100-5639-11e9-9186-b0936a949a47.png)

Reply:
yes, `Create Config Group` button has same locator for both interfaces (when you have table and when 0 entities), so you will be able create it. That is a trick to always create good locators in your code

and Search or Edit or whatever will also work as you get None, not the exception and that is expected behavior. That is guaranteed by `is_searchable` in https://github.com/SatelliteQE/airgun/blob/master/airgun/views/common.py#L414

Reply:
Excellent! good that learn something new today.